cob_hardware_config/cob4-2/config/powerstate_phidget.yaml

```diff
@@ -0,0 +1,5 @@
+voltage_max: 59.5
+voltage_full: 52.0
+voltage_empty: 38.0
```

Comment:
should be 43.0V, see https://github.com/ipa320/cob_robots/pull/514#discussion_r73839576